src/skybox_and_splitscreen.h

```diff
@@ -73,7 +73,7 @@ extern s32 D_800DC5D4;
 extern s32 D_800DC5D8;
 extern u16 D_800DC5BC;
 extern struct GfxPool *gGfxPool;
-extern f32 D_80150130[];
+extern f32 zoomPlayers[];
```

Comment:
When you rename variables. Try to make them only extern'd once; in the file they're declared in. Then every file that uses that variable includes that file.

This can't always be done. But just try once in awhile. it really helps clean the repo up.